src/components/Confirm/index.js

```diff
@@ -163,6 +163,8 @@ class Confirm extends Component  {
   }
 
   onApiSuccess = (apiResponse) => {
+    const duration = Math.round(performance.now() - this.startTime)
+    sendEvent('Upload complete', {duration, method: this.props.method})
```

Comment:
@rfreitas I tried tracking the Spinner screen using the `trackComponent` method. However this did not give accurate timings on woopra (see screenshot). Therefore I think we do need to explicitly time how long the upload takes and send the duration we measure to woopra. I think it makes sense to keep this logic in the Confirm component so I think we should leave this PR how it is.

![screen shot 2017-12-11 at 11 54 44](https://user-images.githubusercontent.com/6737066/33830026-6f7df4b4-de6a-11e7-8561-e179f7a971b5.png)

Reply:
Compare explicitly timing the upload:
![screen shot 2017-12-11 at 11 59 50](https://user-images.githubusercontent.com/6737066/33830138-dfe102a0-de6a-11e7-8b73-720fdcdd1db2.png)